costmap_2d/include/costmap_2d/inflation_layer.h

```diff
@@ -132,8 +132,9 @@ class InflationLayer : public Layer
    * @brief Change the values of the inflation radius parameters
    * @param inflation_radius The new inflation radius
    * @param cost_scaling_factor The new weight
+   * @param inflate_unknown Whether to inflate unknown cells
    */
-  void setInflationParameters(double inflation_radius, double cost_scaling_factor);
+  void setInflationParameters(double inflation_radius, double cost_scaling_factor, bool inflate_unknown);
```

Comment:
The reason that these two parameters were separated is because they are key in computing the cached values. Since inflate_unknown does not affect the cached values, it would be simpler to leave this function definition as it was. Otherwise, it will change the binary compatibility of the object.